pylabrobot/liquid_handling/liquid_handler.py

```diff
@@ -777,8 +778,12 @@ async def aspirate(
 
     # Checks
     for resource in resources:
-      if isinstance(resource.parent, Plate) and resource.parent.has_lid():
-        raise ValueError("Aspirating from a well with a lid is not supported.")
+      if (
+        isinstance(resource.parent, Plate)
+        and resource.parent.has_lid()
+        and not isinstance(resource.parent.lid, Filter)
+      ):
+        raise ValueError("Aspirating from a well with a non-filter lid is not supported.")
```

Comment:
what do you think about making Filter inherit from Resource directly rather than Lid so that this check is not necessary?